### DISCUSSION TITLE
Add future transactions in reports

### DISCUSSION THREAD

**Discussion by lukou31** [firefly-iii/firefly-iii#5018] [Firefly III]
I would like to know if there is a way to add future transactions in reports (operations programed in the automation section).
The idea is to be able to have in a defined timeframe a simluation of the income/outcome including what is going on in the future.

**Comment by JC5** [comment]
There isn't a way to do so.

**Reply by lukou31** [reply]
Ok, but is it something that interest folks and we can expect in a new release ?

I have used Money 99 (I known... it's very old but has some very powerfull functions) until I discover this app. One of those is related to my question and offer a good way to have an estimation of that I can spend for the month after have paid my monthly expenses.

**Reply by JC5** [reply]
https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/#prediction-forecasting-future-gazing
